subsys/bootloader/Kconfig

```diff
@@ -187,6 +187,7 @@ config PM_PARTITION_SIZE_B0_IMAGE
 	default 0x7800 if !B0_MIN_PARTITION_SIZE && (SOC_NRF5340_CPUNET)
 	default FPROTECT_BLOCK_SIZE if SOC_SERIES_NRF91X || SOC_NRF5340_CPUAPP
 	default 0x3800 if SOC_NRF5340_CPUNET
+	default 0x7800 if SOC_NRF54L15_CPUAPP
```

Comment:
the default image in this PR is at 92.5% RRAM usage, not sure if this should be made larger so it fits if zephyr gets bigger in future?

Reply:
this is the last value that satisfies both 4k alignment and maximum of 31k

Reply:
wait, somethings off